src/adk/command_list.py

```diff
@@ -224,9 +228,9 @@ def applications_clone(
         validate_dict = processor.applications_validate(application_name=application_name,
                                                         application_path=application_path)
         if validate_dict["error"] or validate_dict["warning"]:
-            show_validation_messages(validate_dict)
-            typer.echo(f"Local application '{application_name}' is invalid. Application not cloned")
-            return
+            typer.echo("Local application cannot be cloned.")
```

Comment:
This changes the message a bit. The app can be cloned (more or less a copy) when it is validated incorrectly, but we didn't clone it on purpose. We chose only to clone correctly validated apps (after the clone the app is then also valid).

Reply:
Ok, I will fix these messages.